packages/test-cases/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "@web3api/test-cases",
+    "version": "0.0.1-prealpha.1",
+    "license": "MIT",
+    "main": "index.ts",
```

Comment:
private: true

Reply:
this package should also depend on assemblyscript so we can test and execute generated asssemblyscript code